topics:
  input:
    tracks: "/search/tracks"
    rpy: "/anafi/rpy"
    NED_gnss: "/NED_position_drone_t"
  output:
    track_world_coordinate: "/search/tracks/world_coordinates"